galadriel/case/detail.py

```
@@ -53,24 +53,47 @@ def __show_case_as_prerequisite(prerequisite:model.CaseModel):
             rx.table.cell(moment_component.moment(prerequisite.created)),
     )
 
+def __show_empty_case_as_prerequisite(prerequisite:model.CaseModel):
```

Ruff (0.15.6)

[warning] Missing return type annotation for private function `__show_empty_case_as_prerequisite`

(ANN202)